### DISCUSSION TITLE
Docker image update

### DISCUSSION THREAD

**Discussion by SantoshSrinivas79** [Budibase/budibase#3574] [Help] [answered]
Hello,

My docker installation shows `Current version 0.9.187` but the [releases](https://github.com/Budibase/budibase/releases) section is at `v0.9.56`.

The update button in the UI not `budi hosting --update` seem to update.

Please advise. Thank you!

**Comment by shogunpurple** [comment]
Hi @SantoshSrinivas79 - the github releases page does not really get updated - we should automate this. The latest released version of budibase is the one you are on.

**Reply by SantoshSrinivas79** [reply]
@shogunpurple Is there a new docker image update? I am still at `0.9.187`

Other threads show that 1.x is already on!?

**Reply by shogunpurple** [reply]
Yep, just update your installation by going to the "Update" page in the portal.

**Reply by SantoshSrinivas79** [reply]
It isn't working @shogunpurple .. pls advise!

![budibase-update](https://user-images.githubusercontent.com/1036163/144436681-96573036-517b-42ae-ac18-7a21ce85d64d.gif)

**Reply by shogunpurple** [reply]
@SantoshSrinivas79 it seems like your watchtower server may not be functioning properly. The release has definitely been pushed. If you run a `docker ps`, does the watchtower service show up?

If not just try re-running your stack:
```
docker-compose up -d
```

**Reply by SantoshSrinivas79** [reply]
Yes it did show. I tried and updated it. Now its `1.0.5`.
Where can I find the latest release to check against?

Thank you @shogunpurple

**Comment by SantoshSrinivas79** [comment, marked as answer]
```sh
docker-compose up -d
```
